Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -172,7 +172,27 @@ public final class SecureEnclaveValet: NSObject {
             try SecureEnclave.string(forKey: key, withPrompt: userPrompt, options: baseKeychainQuery)
         }
     }
-    
+
+    /// Attempts to retrieve the string at the given key, allowing a custom `fallbackTitle` to be specified to be
+    /// shown to the user in the case that the biometric authentication fails.
+    ///
+    /// - Parameters:
+    ///   - key: A key used to retrieve the desired object from the keychain.
+    ///   - userPrompt: The prompt displayed to the user in Apple's Face ID, Touch ID, or passcode entry UI.
+    ///   - fallbackTitle:  The title of the fallback button shown to the user after 2 failed retrieval attempts.
+    ///                     If the user taps this button, an `LAError.userFallback` will be thrown.
```

Comment:
Can we create our own error here? The fact that we're using an `LAContext` under the hood is something I'd prefer not to expose.

I do like that the name `userFallback` lines up with the `fallbackText` argument name though!

Reply:
Yeah definitely. Should that live in `KeychainError` do you think or a new error type?

Reply:
Per the thread below let's create a new error type.